Source/GTMSessionFetcher.h

```diff
@@ -996,6 +999,13 @@ NSData * GTM_NULLABLE_TYPE GTMDataFromInputStream(NSInputStream *inputStream, NS
 // See comments at the top of this file.
 @property(atomic, copy, GTM_NULLABLE) GTMSessionFetcherRetryBlock retryBlock;
 
+// The optional block for collecting the metrics of the present session.
+//
+// This is called on the callback queue.
+@property(atomic, copy, GTM_NULLABLE)
```

Comment:
These blocks are generally available to set once on the fetcher service, and are propagated to each fetcher created by that service (the client code can always individually set a different one later). Should do that with this as well.

Reply:
Done.